### DISCUSSION TITLE
Store thumbnails in database

### DISCUSSION THREAD

**Discussion by ghost** [filebrowser/filebrowser#1145] [Ideas]
**Is your feature request related to a problem? Please describe.**
filebrowser writes thumbnail images as .jpg files on disk. It may become slow to load and hard to move when the folder gets very large.

**Describe the solution you'd like**
Store thumbnails in a database, like lmdb. Purging the cache will be instantaneous.

**Comment by o1egl** [comment]
I think with that solution images fetching speed will drop significantly

**Comment by ghost** [comment]
@o1egl 

I haven't compared them myself, but it seems storing with LMDB will increase the fetching speed, and there are other benefits.
https://github.com/simon987/sist2/issues/127#issuecomment-726825795
